src/vue/navigation/Passport.vue

```diff
@@ -139,10 +139,6 @@ $dropdown-item-side-padding: 2.4rem;
   display: flex;
   align-items: center;
   justify-content: flex-end;
-
-  @include respond-to-custom($sidebar-hide-bp) {
```

Comment:
why removed?

Reply:
Because Navbar already has these styles so Passport slides to the side.
With removed styles (new version): https://gyazo.com/22e871f18b307cb1d4059a0642406eb2
Without removed styles (old version): https://gyazo.com/6b85d5574dc3efde689b9ea9a3fffd1b